src/main/py/hl7_listener/main.py

```diff
@@ -42,7 +52,11 @@ def exception_formatter(exception_text: str):
 async def process_received_hl7_messages(hl7_reader, hl7_writer, ddspan=None):
     """This will be called every time a socket connects to the receiver/listener."""
     peername = hl7_writer.get_extra_info("peername")
-    logger.info(logging_codes.HL7_MLLP_CONNECTED, peername)
+    logger.info(
+        HL7_MLLP_CONNECTED,
+        logging_code="HL7LLOG002",
+        peername=peername
+    )
```

Comment:
I really hate magic strings, but decided to not go crazy abstracting out the logging codes. For now, I left them as magic strings, but would be happy to move them to the \_\_.init\_\_.py as well, using this pattern:
```
HL7_MLLP_CONNECTED_MESSAGE= "HL7 Listener connection established"
HL7_MLLP_CONNECTED_LOGGING_CODE= "HL7LLOG002"
```

Reply:
See comment above for a thought around encapsulating the 2

Reply:
lean into magic strings, they do wonders when they're only used once :)